symplyphysics/laws/dynamics/total_work_equals_change_in_kinetic_energy.py

```diff
@@ -25,6 +35,64 @@
 law = Eq(total_work, kinetic_energy(time_after) - kinetic_energy(time_before))
 
 
+# Derive the law in case of rectilinear motion with constant total force acting on particle
```

Comment:
Probably also should be integrated (over time, I guess) to remove constant force condition.